zerver/lib/email_notifications.py

```diff
@@ -456,12 +456,17 @@ def do_send_missedmessage_events_reply_in_zulip(
 
     context.update(
         mention="mentioned" in unique_triggers
+        or "topic_wildcard_mentioned" in unique_triggers
         or "stream_wildcard_mentioned" in unique_triggers
+        or "topic_wildcard_mentioned_in_followed_topic" in unique_triggers
         or "stream_wildcard_mentioned_in_followed_topic" in unique_triggers,
```

Comment:
This first block might be better extracted as a separate variable.

Reply:
Done